examples/step_env.py

```diff
@@ -12,7 +21,12 @@
 # Step randomly until interrupted
 try:
     print('Press Ctrl-C to stop...')
+    steps = 0
     while True:
+        if args.n_max_steps:
```

Comment:
hmm just set a default for n_steps, it will make the logic simpler.

Reply:
I was trying to keep the behavior same if example is run without arguments. If I set a default number, it will stop after that number of steps. I could change the argument to '--run_once' with no value though. Did you mean something different?

Reply:
i meant it's fine to change the original behavior a little to make it easier to test.